build.gradle

```diff
@@ -125,7 +125,7 @@ dependencies {
 }
 
 ext {
-    securityPluginVersion = '1.3.1.0'
+    securityPluginVersion = '1.3.2.0'
```

Comment:
It needs only this change to bump security plugin version. The rest of changes are not needed. Please remove them.
If OpenSearch core 1.3.2 is ready, please also update [the version](https://github.com/opensearch-project/security/blob/1.3/build.gradle#L59) too.